security/providers/oidc/src/main/java/io/helidon/security/providers/oidc/DefaultTenantIdProvider.java

```diff
@@ -19,20 +19,19 @@
 import java.util.List;
 import java.util.Optional;
 
+import io.helidon.common.Weight;
 import io.helidon.config.Config;
 import io.helidon.security.ProviderRequest;
 import io.helidon.security.providers.oidc.common.spi.TenantIdFinder;
 import io.helidon.security.providers.oidc.common.spi.TenantIdProvider;
 import io.helidon.security.util.TokenHandler;
 
-import jakarta.annotation.Priority;
-
 import static io.helidon.security.providers.oidc.common.spi.TenantConfigFinder.DEFAULT_TENANT_ID;
 
 /**
  * This is the default tenant id provider.
  */
-@Priority(100000)
+@Weight(100000)
```

Comment:
`Priority(100000)` is very low priority. `Weight(100000)` is very high priority (e.g. it is exactly the opposite meaning).
Is this desired? If you want this to be easily overridden by users, use `Weight(1)`